### PR TITLE
Feat #6382 part 2

### PR DESCRIPTION
**Pull request by phpgeekfr**
Feat #6382 part 2

### CODE REVIEW

File: libs/organization/src/lib/organization/pages/request-access-pending/request-access-pending.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<app-logo (click)="logout()"></app-logo>
+
+<mat-card fxLayout="column" fxLayoutAlign="center center">
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/pages/request-access-pending/request-access-pending.component.html:3]: If you're not using mat-card directive (mat-card-image, mat-card-title, ...) then do not use mat-card. Use the class="surface" instead.

File: libs/organization/src/lib/organization/pages/request-access-pending/request-access-pending.component.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+mat-card {
+  text-align: center;
+  padding-top: 48px;
+  margin: auto;
+  width: 50%;
+  
+  img {
+    height: auto;
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/pages/request-access-pending/request-access-pending.component.scss:26]: remove this height, except if you know exactly the ratio of the image, then set the exact value for height & width. That will avoid potential ump as th browser will reserve this space for the img before fetching it.

File: libs/organization/src/lib/organization/pages/request-access/request-access.component.html [hunk 3]
```diff
@@ -1,15 +1,39 @@
...
-  <p class="mat-body-1">
-    We noticed that your email address is associated with an existing user account of one of our applications.<br/>
-    To access Archipel Content, please request access and wait for our team's approval.
-  </p>
-  <p class="mat-body-2">N.B.: All members from your company will also get granted access to {{ appName }}.</p>
-  <button mat-flat-button color="primary" (click)="requestAccess()" [disabled]="disabledRequest">Request Access</button>
-  <button mat-button color="primary" (click)="refresh()">Refresh Page</button>
+  <ng-container *ngIf="step$ | async as step">
+
+    <ng-container *ngIf="step === 'initial'">
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/pages/request-access/request-access.component.html:6]: Maybe using `ngSwitchCase` make more sense here. But it's not super important

File: libs/organization/src/lib/organization/pages/request-access/request-access.component.html [hunk 4]
```diff
@@ -1,15 +1,39 @@
...
+    </ng-container>
+
+    <ng-container *ngIf="step === 'request'">
+      <h1>What access are your looking for ?</h1>
+      <p class="mat-body-1">
+        To access {{ appName[currentApp] }}, please select a "role". It will determine the features you can acccess once you've joined the platform.
+      </p>
+      <p class="mat-body-2">N.B.: All members from your company will be granted the same access.</p>
+
+      <article fxLayout="row" fxLayoutAlign="space-between center" fxLayout.lt-md="column">
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/pages/request-access/request-access.component.html:23]: you don't nee to specify `fxLayout="row"` as `row` is the default. You can do `fxLayout`

File: libs/organization/src/lib/organization/pages/request-access/request-access.component.html [hunk 5]
```diff
@@ -1,15 +1,39 @@
...
+        <div fxLayout="row">
+          <p>Choose your role * :</p>
+          <mat-icon svgIcon="info"
+            matTooltip="Depending on your role, you might have access to different features of the platform."></mat-icon>
+        </div>
+        <mat-button-toggle-group [ngClass]="{ disabled: false }" name="appAccess" [formControl]="formControl">
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/pages/request-access/request-access.component.html:29]: Hmmm I don't understand why you force the element to NOT have the class `disabled`?

File: libs/organization/src/lib/organization/pages/request-access/request-access.component.html [hunk 6]
```diff
@@ -1,15 +1,39 @@
...
+
+    <ng-container *ngIf="step === 'request'">
+      <h1>What access are your looking for ?</h1>
+      <p class="mat-body-1">
+        To access {{ appName[currentApp] }}, please select a "role". It will determine the features you can acccess once you've joined the platform.
+      </p>
+      <p class="mat-body-2">N.B.: All members from your company will be granted the same access.</p>
+
+      <article fxLayout="row" fxLayoutAlign="space-between center" fxLayout.lt-md="column">
+        <div fxLayout="row">
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/pages/request-access/request-access.component.html:24]: I think a `label` would make more sense here : 
```html
<label>
  <span>Choose your role * :</span>
  <mat-icon ...></mat-icon>
</label>
```
You would also benefit from the global spacing between span & mat-icon like that.

File: libs/organization/src/lib/organization/pages/request-access/request-access.component.scss [hunk 7]
```diff
@@ -36,4 +37,19 @@ mat-card {
...
+  margin-bottom: 8px;
+}
+
+mat-button-toggle {
+  min-width: 125px;
+}
+
+mat-icon {
+  margin-left: 4px;
+  margin-right: 12px;
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/pages/request-access/request-access.component.scss:50]: I think this part of the style should be on the label (see my comment above) : 
```css
label {
  padding: 0 12px;
}
```

**Review by GrandSchtroumpf — APPROVED** [review]

File: libs/organization/src/lib/organization/pages/request-access/request-access.component.html (hunk 5 above)
Comment (phpgeekfr) [reply libs/organization/src/lib/organization/pages/request-access/request-access.component.html:29]: bad copy/paste ..

**Pull request merged**